src/lib/swapper/swappers/ThorchainSwapper/utils/longTailHelpers.ts

```diff
@@ -107,54 +112,62 @@ type ContractData = {
   tokenOut: Address
 }
 
-export const getContractDataByPool = async (
-  poolAddresses: Map<Address, FeeAmount>,
+export const getContractDataByPool = (
+  poolAddresses: Map<Address, { token0: Address; token1: Address; fee: FeeAmount }>,
```

Comment:
`token0Address` `token1Address` vernacular to distinguish from `token0` as a `Token`?

Reply:
🧠  https://github.com/shapeshift/web/pull/5902/commits/4bde8e1a0bb1460511b5d4254c174975d04efeb4